MANIFEST.in

```diff
@@ -1 +1,3 @@
 include pisa/oscillations/prob3/*.h
+include pisa/utils/gaussians.pyx
+include pisa/utils/gaussians.c
```

Comment:
@sboeser @jllanfranchi  In this PR, gaussians.c is not included, even though it is shown here. Based on the discussion on the call today, what we (Justin) will do is remove the compilation stage of the gaussians module and we'll simply make cython an optional package to install and use for this stage (and for future pisa development if desired).